user/enterprise/platform-tips.md

```diff
@@ -309,6 +309,28 @@ Additionally, please the following command to clean up all Docker build images:
 $ sudo docker images | grep travis | awk '{print $3}' | xargs sudo docker rmi -f
 ```
 
+## Find out maximum available concurrency
+
+To find out how much concurrency is available in your Travis CI Enterprise setup, connect to your platform machine via ssh and run:
+
+```
+$ travis bash
+root@te-main:/# rabbitmqctl list_consumers -p travis | grep builds.trusty | wc -l
+```
+
+The number that's returned here is equal to the maximum number of concurrent jobs that are available. To adjust concurrency, please follow the instructions [here](/user/enterprise/worker-configuration/#configuring-the-number-of-concurrent-jobs) for each worker machine.
+
+## Find out how many worker machines are connected
+
+If you wish to find out how many worker machines are currently connected, please connect to your platform machine via ssh and follow these steps:
+
+```
+$ travis bash
+root@te-main:/# rabbitmqctl list_consumers -p travis | grep amq.gen- | wc -l
+```
+
+If you need to spin up more worker machines, please see our docs about [installing new worker machines](/user/enterprise/installation/#install-travis-ci-worker).
```

Comment:
We *probably* should not use spin up ? There are two other uses, both in enterprise. Maybe use "start"? or "boot"?